ast/builtins.go

```diff
@@ -2765,6 +2774,7 @@ var netCidrContainsMatchesOperandType = types.NewAny(
 	)),
 )
 
+// Marked non-deterministic because DNS resolution results can be non-deterministic.
```

Comment:
Ugh this might trip up the linter... It doesn't start with `// NetLookupIPAddr ...`. But I think I had expected that before and was wrong about it; so if this goes through CI, fine.

Reply:
It looks like it went through CI okay...